requirements.txt

```diff
@@ -1,37 +1,60 @@
+alabaster==0.7.12
 appdirs==1.4.4
 atomicwrites==1.4.0
 attrs==20.2.0
+Babel==2.8.0
 black==20.8b1
 certifi==2020.6.20
 chardet==3.0.4
 click==7.1.2
 colorama==0.4.3
 coverage==5.2.1
+docutils==0.16
+flake8==3.8.3
 Flask==1.1.2
 ibm-cloud-sdk-core==1.7.3
 ibm-watson==4.7.1
```

Comment:
I had to run pip install flask_session to get it working in addition to pip install -r requirements.txt

Reply:
Will fix